File: src/cn/langya/sun/modules/Category.java
package cn.langya.sun.modules;

public enum Category {
    Combat("Combat"),
    Move("Move"),
    World("World"),
    Render("Render"),
    Client("Client"),
    Misc("Misc"),
    Player("Player");

    private final String displayName;

    Category(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }
}

File: src/cn/langya/sun/modules/Category.kt
package cn.langya.sun.modules

/**
 * @author LangYa
 * @ClassName Module
 * @date 2023/12/14 21:10
 * @Version 1.0
 */

enum class Category(val displayName: String) {
    Combat("战斗"),
    Move("移动"),
    Render("视觉"),
    Client("客户端"),
    Misc("其他"),
}